src/style.css

```
@@ -0,0 +1,74 @@
+.stars_item {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.stars {
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-start;
+  min-height: 16px;
+  background-color: #fff;
+  background-repeat: no-repeat,
+    no-repeat,
+    no-repeat,
+    no-repeat,
+    no-repeat;
+  background-position: left top,
+    20px top,
+    40px top,
+    60px top,
+    80px top;
```

```suggestion
  background-repeat: no-repeat,
  background-position: 50%,
```
Задай для stars__star типові стилі для неактивної зірки, а потім через :nth-child та модифікатори додавай background-image для необхідної кількості зірок, наприклад:
```
.stars--4 > :nth-child(-n+4),
.stars--5 > :nth-child(-n+5) {
  background-image: url(images/star-active.svg);
}
```